### DISCUSSION TITLE
🌟 v0.3.0 - Template clean up and updates, multi-language support

### DISCUSSION THREAD

**Discussion by ptsouchlos** [DeveloperPaul123/modern-cv#23] [Announcements]
## What's Changed
* 0.2.0 release changes (back merge) by @DeveloperPaul123 in https://github.com/DeveloperPaul123/modern-cv/pull/13
* Template improvements, bug fixes and style updates by @DeveloperPaul123 in https://github.com/DeveloperPaul123/modern-cv/pull/20
* Ignore contact fields that are set to none  by @caemor in https://github.com/DeveloperPaul123/modern-cv/pull/16
* Add multi-langauge support by @DeveloperPaul123 in https://github.com/DeveloperPaul123/modern-cv/pull/22

## New Contributors
* @caemor made their first contribution in https://github.com/DeveloperPaul123/modern-cv/pull/16

**Full Changelog**: https://github.com/DeveloperPaul123/modern-cv/compare/0.2.0...0.3.0

<hr /><em>This discussion was created from the release <a href='https://github.com/DeveloperPaul123/modern-cv/releases/tag/0.3.0'>🌟 Template clean up and updates, multi-language support</a>.</em>

**Comment by 9SMTM6** [comment]
This doesn't yet seem to be live on typst.app/universe.

Totally okay if that is expected, just wanted to let you know in case it was not intended.

Typst compiler as packaged on arch thinks the 0.3.0 package doesn't exist, the vscode lsp mentiones a failure to decompress - but it also cant handle the 0.1.0 release, so its probably not correct. Did not yet test on typst.app, my internet is currently unreliable.

**Reply by Joelius300** [reply]
The PR for the update is open, unfortunately it hasn't been merged yet. See https://github.com/typst/packages/pull/555

**Reply by 9SMTM6** [reply]
Thanks, should've guessed that this requires PRs on typst side too

**Reply by ptsouchlos** [reply]
Thanks for the quick reply @Joelius300 !

**Reply by ptsouchlos** [reply]
Hoping to get it merged today :)

**Reply by ptsouchlos** [reply]
`0.3.0` release was merged today, should be live later today or tomorrow on typst universe.

**Reply by 9SMTM6** [reply]
Works!
